deps/build_FLINT.v200.690.0.jl

```diff
@@ -0,0 +1,48 @@
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
+
+# Parse some basic command-line arguments
+const verbose = "--verbose" in ARGS
+const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
+products = [
+    LibraryProduct(prefix, ["libflint"], :libflint),
+]
+
+# Download binaries from hosted location
+bin_prefix = "https://github.com/JuliaBinaryWrappers/FLINT_jll.jl/releases/download/FLINT-v200.690.0+0"
+
+# Listing of files generated by BinaryBuilder:
+download_info = Dict(
+    Linux aarch64 {libc=glibc} => ("$bin_prefix/FLINT.v200.690.0.aarch64-linux-gnu.tar.gz", "4a8aa3d405223e7334e9a53e4dde37455edc1431551e8f12102f3cf0835e0f28"),
```

Comment:
The keys are wrong here. I am guessing platforms are not serialized correctly? Anyway, we need to match what works in Julia 1.0-1.2...
```suggestion
    Linux(:aarch64, libc=:glibc) => ("$bin_prefix/FLINT.v200.690.0.aarch64-linux-gnu.tar.gz", "4a8aa3d405223e7334e9a53e4dde37455edc1431551e8f12102f3cf0835e0f28"),
```

Reply:
Working on it.